cmd/cg-client/main.go

```diff
@@ -397,6 +398,20 @@ func main() {
 	if *baseURL != "" {
 		config.BaseURL = *baseURL
 	}
+	// Because the want to display a sensible error message when using flags
```

Comment:
nit: Because _we_ want (based on context)

Reply:
fixed. Thank you